### ISSUE TITLE
finetune single speaker

### ISSUE THREAD

**Issue by kunibald413** [k2-fsa/ZipVoice#13]
Could you explain if it's possible to fine-tune single speaker?

Thank you!

**Comment by zhu-han** [comment]
Yes, you can fine-tune it on single speaker. Just use a smaller learning rate to fine-tune the provided checkpoints. Note that you'll still need to use the target speaker's prompt-wav and prompt-text in inference.

**Comment by tujie-jiangye** [comment]
@zhu-han I want to finetune on a single speaker and have about 10 minutes of data. Could you please provide a suitable set of parameters

**Comment by Mrjack0912** [comment]
Hello, relate to single speaker fine-tuning. if in runtime i want to use distilled model only, which solution should be better:
1. use fine-tuned checkpoint as teacher model to distill with single speaker data (run from stage 4-6 like  in run_emilia.sh)
2. use normal checkpoint as teacher model  then just distill with single speaker data
